feature-libs/product-configurator/common/shared/utils/common-configurator-utils.service.ts

```diff
@@ -103,4 +103,33 @@ export class CommonConfiguratorUtilsService {
     });
     return numberOfIssues;
   }
+
+  /**
+   * Verifies whether the configurator type is an attribute based one.
+   *
+   * @param {string} configuratorType - Configurator type
+   * @returns {boolean} - 'True' if the expected configurator type, otherwise 'false'
+   */
+  isAttributeBasedConfigurator(configuratorType: string | undefined): boolean {
+    if (configuratorType) {
+      return (
+        configuratorType === 'CPQCONFIGURATOR' ||
```

Comment:
For type safety, can we create an enum with 3 configurator types and use it consistently everywhere (as values and as type)? Naming is up to you. What do you think about keys of enum: CPQ, VARIANT, TEXTFIELD?

Reply:
Agree

Reply:
Done